src/cljs/salttoday/pages/common.cljs

```diff
@@ -15,6 +15,9 @@
 
 (defn make-layout
   [current-page content]
-  [:div.outer [:div.navigation [nav-link "#/" "Home " current-page :home "icon-home3"] [nav-link "#/users" "Users " current-page :users "icon-group"]]
-   [:div [:img.title-image {:src "/img/soo-salty.png"}] content]
-   [:img.footer-pile {:src "/img/footer-pile.png"}]])
+  [:div.outer
+  [:div.content-wrapper [:div.navigation [nav-link "#/" "Home " current-page :home "icon-home3"] [nav-link "#/users" "Users " current-page :users "icon-group"]]
```

Comment:
this should be indented since it's inside another div

Reply:
I think it follows the format now